README.md

```diff
@@ -58,7 +58,7 @@ After successful deployment, you can invoke the deployed function by using the f
 serverless invoke --function main
 ```
 
-## Local development
+## 🏗️ Local development
```

Comment:
Sorry @inigomarquinez my comment was misplaced. The guide I was asking about is related to this environment variables. Where can all these variables be retrieved from?

Reply:
@totegsito , I've referenced this in a new issue to improve the documentation: https://github.com/guidesmiths/bamboohrbot/issues/29